README.md

```diff
@@ -5,21 +5,13 @@ Docker images for deploying and running the Confluent Platform.  The images are
 
 Full documentation for using the images can be found [here](http://docs.confluent.io/current/cp-docker-images/docs/intro.html).
 
-# Important Caveat - Images Not Supported for Docker for Mac
```

Comment:
Not really a big deal, I'm just surprised because I didn't think the issues with Docker for Mac had gone away.